### DISCUSSION TITLE
Technical Studentship at CERN

### DISCUSSION THREAD

**Discussion by nyrareddy** [fastmachinelearning/hls4ml#857] [General]
As I prepare my application for the upcoming technical studentship at CERN, I wanted to inquire about the possibility of hls4ml projects being included in this year's program. Additionally, I am curious if any projects have already been selected for this year's studentship. Are there any prequisites for being successful in completing these projects?

**Comment by pierinim** [comment]
Hello

we didn't select technical students to work in hls4ml since more than a year, when ERC funds ran out. hls4ml (so far) didn't receive CERN funds. We open positions when funds become available. We are most likely opening one to start soon, to work on a satellite application (we received funds from the EU for a project we submitted). In your application, make it clear that you are interested, using words like hls4ml, FPGA, etc in your application. We don't read each individual application so you have to make your visible for us to consider it. 

Good luck

**Comment by nyrareddy** [comment]
> We are most likely opening one to start soon, to work on a satellite application (we received funds from the EU for a project we submitted)

Where can I find this opening? Are there any tentative timelines for this?

> In your application, make it clear that you are interested, using words like hls4ml, FPGA, etc in your application. We don't read each individual application so you have to make your visible for us to consider it.

Will hls4ml still be a possible project in this CERN Technical studentship?

**Reply by pierinim** [reply]
We are = we didn’t yet.

we might have something out in October to start in Jan. We will advertise it here
